package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@gglukmann/use-google-sheets",
   "description": "### A React Hook wrapper library for google-sheets-mapper for getting data from Google Sheets API v4",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Can you remove the version here? I'll create a new version after is merge this PR.

Reply:
Yes, done it in the last commits !